eras/shelley/impl/src/Cardano/Ledger/Shelley/TxCert.hs

```diff
@@ -211,12 +212,12 @@ instance Era era => EncCBOR (ShelleyTxCert era) where
 
 encodeShelleyDelegCert :: Crypto c => ShelleyDelegCert c -> Encoding
 encodeShelleyDelegCert = \case
-  RegKey cred ->
+  ShelleyRegCert cred ->
     encodeListLen 2 <> encodeWord8 0 <> encCBOR cred
-  DeRegKey cred ->
+  ShelleyUnRegCert cred ->
```

Comment:
I think `UnReg` is better than `DeReg`, since `De` is also the start of `Delegation` , but there's no ambiguity with `UnReg`   
(I guess unless you are reading this in French as "A Registration" :laughing:  )